### DISCUSSION TITLE
Is it possible to use DrizzleORM with this plugin?

### DISCUSSION THREAD

**Discussion by scr2em** [capacitor-community/sqlite#443] [General]
here is the docs 
https://orm.drizzle.team/docs/installation-and-db-connection

**Comment by jepiqueau** [comment]
@scr2em It is not planned

**Reply by scr2em** [reply]
@jepiqueau sad news, thanks for all your efforts.

**Comment by jepiqueau** [comment]
@scr2em you can write yourself a driver for @capacitor-community/sqlite and share it to the community

**Reply by scr2em** [reply]
@jepiqueau I'm trying but I still have some difficulties to connect the dots in the plugin jeep-sqlite, ios and android parts. perhaps you can give me a quick 15mins onboarding ?

**Reply by oliveryasuna** [reply]
@scr2em This may help: https://github.com/DawidWetzler/capacitor-sqlite-kysely
